src/iconsManifest/languages.ts

```diff
@@ -154,7 +154,7 @@ export const languages: ILanguageCollection = {
   qsharp: { ids: 'qsharp', defaultExtension: 'qs' },
   r: { ids: 'r', defaultExtension: 'r' },
   racket: { ids: 'racket', defaultExtension: 'rkt' },
-  razor: { ids: 'razor', defaultExtension: 'cshtml' },
+  razor: { ids: 'aspnetcorerazor', defaultExtension: 'cshtml' },
```

Comment:
According to https://github.com/Microsoft/vscode/blob/1.29.1/extensions/razor/package.json#L15 the `language id` is still `razor`. If you want to add `aspnetcorerazor`, which I guess is of another extension, you should switch `ids` to an `array` that contains both `ids`. (i.e. `ids: ['razor', 'aspnetcorerazor']`).

Reply:
I changed the vscode git to include a aspnetcorerazor alias term in the razor id.

Reply:
Again. What you've done is not the proper way. Close the PR at `vscode`, and follow my instructions. Don't try to re-invent the wheel.

Reply:
Sorry, I misunderstood. You are saying I should change the above PR to reference razor from vscode? Sorry, I'm (obviously) new to this.

Reply:
Bingo. I already gave you the code snippet that you need to use.